### PR TITLE
[DROOLS-2075] Allow empty output cell on DTs

### PR DESCRIPTION
**Pull request by kurobako**
https://issues.jboss.org/browse/DROOLS-2075

### CODE REVIEW

File: kie-dmn/kie-dmn-feel/src/main/java/org/kie/dmn/feel/lang/impl/CompiledExpressionImpl.java [hunk 1]
```diff
@@ -38,6 +38,7 @@ public void setExpression( ASTNode expression ) {
     }
 
     public Object evaluate(FEELEventListenersManager eventsManager, Map<String, Object> inputVariables) {
+        if (expression == null) return null;
```
Comment (jomarko) [inline kie-dmn/kie-dmn-feel/src/main/java/org/kie/dmn/feel/lang/impl/CompiledExpressionImpl.java]: This is just formatting issue, please would you mind rewrite as:
```java
if (expression == null) { 
    return null;
}
```

File: kie-dmn/kie-dmn-core/src/test/java/org/kie/dmn/core/DMNDecisionTableRuntimeTest.java [hunk 2]
```diff
@@ -320,4 +320,17 @@ public void testDTUsingEqualsUnaryTestWithVariable2() {
...
+    public void testEmptyOutputCell() {
+        final DMNRuntime runtime = DMNRuntimeUtil.createRuntime( "DT_empty_output_cell.dmn", this.getClass() );
+        final DMNModel dmnModel = runtime.getModel( "http://www.trisotech.com/definitions/_77ae284e-ce52-4579-a50f-f3cc584d7f4b", "Calculation1" );
+        assertThat( dmnModel, notNullValue() );
+        final DMNContext context = DMNFactory.newContext();
+        context.set( "MonthlyDeptPmt", BigDecimal.valueOf( 1 ) );
+        context.set( "MonthlyPmt", BigDecimal.valueOf( 1 ) );
+        context.set( "MonthlyIncome", BigDecimal.valueOf( 1 ) );
+        final DMNResult dmnResult = runtime.evaluateAll( dmnModel, context );
+        assertThat( dmnResult.hasErrors(), is( false ) );
```
Comment (jomarko) [inline kie-dmn/kie-dmn-core/src/test/java/org/kie/dmn/core/DMNDecisionTableRuntimeTest.java:335]: @kurobako, sorry I didn't noticed in first round, would you mind add one assertion like bellow? I think such assert should confirm a little bit more that the issue is fixed. Let me know what do you think.
```java
assertNull( dmnResult.getContext().get("Logique de décision 1") );
```

Comment (kurobako) [reply kie-dmn/kie-dmn-core/src/test/java/org/kie/dmn/core/DMNDecisionTableRuntimeTest.java:335]: @jomarko yeah, why not, I see no problem with it

**Review by jomarko — APPROVED** [review]
Thanks @kurobako , approving.

**Comment by etirelli** [comment]
@kurobako sorry, can you please rebase the PR. It looks like there is a conflict with another commit.

**Comment by kurobako** [comment]
@etirelli that's ok, updated

**Pull request merged**